exec/jdbc/src/main/java/org/apache/drill/jdbc/SqlTimeoutException.java

```diff
@@ -23,12 +23,17 @@
  * Indicates that an operation timed out. This is not an error; you can
  * retry the operation.
  */
-public class SqlTimeoutException
-    extends SQLException
-{
+public class SqlTimeoutException extends SQLException {
+
+  private static final long serialVersionUID = 2017_06_20L;
```

Comment:
maybe use -1  or use your IDE serialVersionUID generator (I know there's an algorithm for it...), but I don't think Drill is making any effort to make this class serializable so it could be a `@SuppressedWarning` annotation as well.